lib/routes/tsinghua/lib/tzgg.ts

```diff
@@ -0,0 +1,73 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    path: '/lib/tzgg/:category',
+    categories: ['university'],
+    example: '/tsinghua/lib/tzgg/qtkx',
+    parameters: { category: '分类，必须填写，可在对应分类页 URL 中找到' },
```

Comment:
```suggestion
    parameters: { category: '分类，可在对应分类页 URL 中找到' },
```
No need to mention the necessity, it will be populated automatically from https://github.com/DIYgod/RSSHub/blob/360b47fa20cf7427775f2fd4e364349e0c72bc94/lib/routes/tsinghua/lib/tzgg.ts#L7